src/locales/en/messages.json

```diff
@@ -3562,6 +3562,21 @@
   "personalOwnershipSubmitError": {
     "message": "Due to an Enterprise Policy, you are restricted from saving items to your personal vault. Change the Ownership option to an organization and choose from available Collections."
   },
+  "disableSend": {
+    "message": "Disable Send"
```

Comment:
Each of these needs:
`"description": "'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated."`